index.js

```diff
@@ -367,11 +399,13 @@ function browserslist (queries, opts) {
     name1 = name1.split(' ')
     name2 = name2.split(' ')
     if (name1[0] === name2[0]) {
-      if (FLOAT_RANGE.test(name1[1]) && FLOAT_RANGE.test(name2[1])) {
-        return parseFloat(name2[1]) - parseFloat(name1[1])
-      } else {
-        return compare(name2[1], name1[1])
-      }
+      // here we assume that caniuse version ranges never overlaps,
+      // so it is safe to use the left of the range
+      // eslint-disable-next-line max-len
```

Comment:
Nope, you can't just disable the rule. Try to reorganize the code to make it more readable.